test/typedarray/rlexe.xml

```diff
@@ -215,13 +215,13 @@ Below test fails with difference in space. Investigate the cause and re-enable t
   <test>
     <default>
       <files>allocation.js</files>
-      <tags>typedarray,exclude_arm</tags>
+      <tags>typedarray,exclude_arm,xplatslow</tags>
```

Comment:
The reason these tests were slow in fact due to OOM fixed by https://github.com/Microsoft/ChakraCore/pull/1086 .At this point I don't see a reason to disable these.